### ISSUE TITLE
please update your video on youtube

### ISSUE THREAD

**Issue by csfreebird** [Kungsgeten/org-brain#85]
There is a tutorial video on YouTube based on an out-of-date version of org-brain, and the tutorial will at first be quite confusing.

**Comment by Kungsgeten** [comment]
Yes I'm aware of that. I've been working on an "export to HTML" feature for `org-brain`, and I'd like to have a basic version of that before posting a new tutorial. I should probably add a comment on the YouTube-video that it is outdated though...

**Comment by Louis-Cypher** [comment]
I'd like to strongly support that. I started using org-brain today and for me the tutorial was more confusing than useful. It starts with an org-brain statement that is central (org-brain-open)   but doesn't exist anymore, some configuration that doesn't exist any more. 

I tried to figure out how org-brain works using the video over an hour and would have been much faster in case I would have only used the Readme file. I think it would help your amazing plugin a lot to put some effort in documentation/ tutorials. I'd love to help but I am not yet capable of doing so
